Vic2ToHoI4/Source/V2World/States/State.cpp

```diff
@@ -53,6 +53,26 @@ int Vic2::State::determineEmployedWorkersScore(const workerStruct& workers)
 }
 
 
+void Vic2::State::eatState(const State& state, const StateDefinitions& stateDefinitions)
```

Comment:
I get merging of provinces but what does this do for locs? Say you have a vic2 "English Botswana" and "French Botswana", what will happen after merge?

Reply:
Names are determined when creating HoI4 states from Vic2 states. Because this is done before that point, it Just Works (tm).